.github/workflows/jmh-benchmark.yml

```diff
@@ -0,0 +1,101 @@
+name: JMH Benchmark
+
+on:
+  issue_comment:
+    types: [created]
+
+
+jobs:
+  setup:
+    if: github.event.issue.pull_request && contains(github.event.comment.body, '/benchmark')
+    runs-on: ubuntu-latest
+    permissions: write-all
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Get PR branch
+      uses: xt0rted/pull-request-comment-branch@v1
+      id: comment-branch
+
+
+    - name: Set branch name
+      run: echo "BRANCH_NAME=${{ steps.comment-branch.outputs.head_ref }}" >> $GITHUB_ENV
+    
+    - id: 'auth'
+      name: Authenticating
+      uses: 'google-github-actions/auth@v1'
+      with:
+        credentials_json: '${{ secrets.GCP_SA_KEY }}'
+
+    - name: Set up Google Cloud SDK
+      uses: google-github-actions/setup-gcloud@v1
+      
+      
+    - name: Setup Directory and Copy Bash Scripts
+      run: |
+        gcloud compute ssh \
+          --project=nullway-jmh \
+          --zone=us-central1-a \
```

Comment:
Do we need to hardcode the zone?

Reply:
It is required for using ssh. Can use a service account instead to avoid this, happy to look into that if that is the approach we want to take.